README.md

```diff
@@ -200,11 +200,9 @@ Copyright:
  © 2013-2018 Alexey Khokholov (Nuke.YKT).  
 License: [GPL-2.0+](https://www.gnu.org/licenses/old-licenses/gpl-2.0.html)
 
-Files: `soundfonts/TimGM6mb.sf2`  
+Files: `soundfonts/GeneralUser_GS_v1.511.sf3`  
 Copyright:  
- © 2004 Tim Brechbill;  
- © 2010 David Bolton.  
-License: [GPL-2.0](https://www.gnu.org/licenses/old-licenses/gpl-2.0.html)
+ © 2020 S. Christian Collins
```

Comment:
Please do at least point to the local copy of the custom (*sigh!*) license and call it "GeneralUser GS v1.51 License v2.0" or similar.

Reply:
How about "public domain"? In fact, it is.

Reply:
No, wait, public domain needs to be explicitly declared as such.